### PR TITLE
make jira tag for feedback more descriptive

### PR DESCRIPTION
**Pull request by InsaneZein**
for this jira: [RHCLOUD-18880](https://issues.redhat.com/browse/RHCLOUD-18880)

### CODE REVIEW

File: src/js/App/Feedback/Feedback.js [hunk 1]
```diff
@@ -29,7 +37,7 @@ const Feedback = ({ user }) => {
         },
         body: JSON.stringify({
           description: `Feedback: ${textAreaValue}, Username: ${user.identity.user.username}, Account ID: ${user.identity.account_number}, Email: ${user.identity.user.email}, URL: ${window.location.href}`, //eslint-disable-line
-          summary: `${window.insights.chrome.isProd ? '[PROD]' : '[PRE-PROD]'} App Feedback`,
+          summary: `${addFeedbackTag()} App Feedback`,
```
Comment (ryelo) [inline src/js/App/Feedback/Feedback.js:40]: can we put the brackets around this so it comes up as `[Insights] App Feedback`?

**Review by ryelo — CHANGES_REQUESTED** [review]

File: src/js/App/Feedback/Feedback.js (hunk 1 above)
Comment (InsaneZein) [reply src/js/App/Feedback/Feedback.js:40]: oh good catch

File: src/js/App/Feedback/Feedback.js [hunk 2]
```diff
@@ -18,6 +18,14 @@ const Feedback = ({ user }) => {
   const isAvailable = env === 'prod' || env === 'stage';
   const setIsModalOpen = (...args) => dispatch(toggleFeedbackModal(...args));
 
+  const addFeedbackTag = () => {
```
Comment (Hyperkid123) [inline src/js/App/Feedback/Feedback.js:21]: Lets make it a bit less wordy: 
```js
const addFeedbackTag = () => window.insights.chrome.isProd ? `[${bundle}]` : '[PRE-PROD]'
```

File: src/js/App/Feedback/Feedback.js [hunk 3]
```diff
@@ -18,6 +18,14 @@ const Feedback = ({ user }) => {
...
+    if (window.insights.chrome.isProd) {
```
Comment (Hyperkid123) [inline src/js/App/Feedback/Feedback.js:22]: Can you create a helper function that will give you the same value? We want to stop using the window API.

You can check this file to get the implementation: https://github.com/RedHatInsights/insights-chrome/blob/master/src/js/chrome/entry.js#L112

Comment (InsaneZein) [reply src/js/App/Feedback/Feedback.js]: Hmm I'm a bit confused by this. Looking at the implementation of that file, getApp is also using the window API.

Comment (Hyperkid123) [reply src/js/App/Feedback/Feedback.js]: Yup, there is a plenty of parts that are using the window. We want to migrate away from it.

Comment (InsaneZein) [reply src/js/App/Feedback/Feedback.js]: I've changed the implementation. Let me know if this is what you meant.

**Review by Hyperkid123 — APPROVED** [review]

File: src/js/App/Feedback/Feedback.js [hunk 4]
```diff
@@ -6,6 +6,7 @@ import Cookies from 'js-cookie';
 import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import { toggleFeedbackModal } from '../../redux/actions';
+import { isProd } from '../utils.ts';
```
Comment (Hyperkid123) [inline src/js/App/Feedback/Feedback.js:9]: This should be 
```js
import { isProd } from '../../utils.ts';
```

**Pull request merged**